### DISCUSSION TITLE
Looking for all CORMAS contributors

### DISCUSSION THREAD

**Discussion by SergeStinckwich** [cormas/cormas#772] [General]
This would be nice to have a list of all the CORMAS contributors: not only current main developers, but also previous developers, testers, models designers, etc ...

I install the [All Contributors](https://allcontributors.org/) bot in order to automate the process. See one example below.

**Comment by SergeStinckwich** [comment]
@all-contributors please add @hernanmd  for code and test

**Comment by allcontributors** [comment]
@SergeStinckwich 

I've put up [a pull request](https://github.com/cormas/cormas/pull/219) to add @hernanmd! :tada:

**Comment by tomooda** [comment]
@all-contributors please add @SergeStinckwich  for code and test

**Comment by allcontributors** [comment]
@tomooda 

I've put up [a pull request](https://github.com/cormas/cormas/pull/220) to add @SergeStinckwich! :tada:

**Comment by SergeStinckwich** [comment]
@all-contributors please add @npapoylias for code and test

**Comment by allcontributors** [comment]
@SergeStinckwich 

I've put up [a pull request](https://github.com/cormas/cormas/pull/221) to add @npapoylias! :tada:

**Comment by SergeStinckwich** [comment]
@all-contributors please add @pierbo for code

**Comment by allcontributors** [comment]
@SergeStinckwich 

I've put up [a pull request](https://github.com/cormas/cormas/pull/222) to add @pierbo! :tada:

**Comment by allcontributors** [comment]
@SergeStinckwich 

I've updated [the pull request](https://github.com/cormas/cormas/pull/222) to add @pierbo! :tada:

**Comment by SergeStinckwich** [comment]
@all-contributors please add @ChLePage  for code

**Comment by allcontributors** [comment]
@SergeStinckwich 

I've put up [a pull request](https://github.com/cormas/cormas/pull/223) to add @ChLePage! :tada:

**Comment by SergeStinckwich** [comment]
@all-contributors please add @cdrick65 for code

**Comment by allcontributors** [comment]
@SergeStinckwich 

I've put up [a pull request](https://github.com/cormas/cormas/pull/225) to add @cdrick65! :tada:

**Comment by SergeStinckwich** [comment]
@all-contributors please add @nbecu for code

**Comment by allcontributors** [comment]
@SergeStinckwich 

I've put up [a pull request](https://github.com/cormas/cormas/pull/226) to add @nbecu! :tada:

**Comment by SergeStinckwich** [comment]
@all-contributors please add @brun0 for code

**Comment by allcontributors** [comment]
@SergeStinckwich 

I've put up [a pull request](https://github.com/cormas/cormas/pull/227) to add @brun0! :tada:

**Comment by SergeStinckwich** [comment]
found this list in the CORMAS code, that I guess should be added to the contributors list. Do you know others?

- Innocent Bakam
- François Bousquet
- Alassane Bah
- Emmanuel Lieurain
- Jean-Christophe Soulié
- Paul Guyot
- Jean-François Lefevre
- Anne Dray
- Géraldine Abrami (INRAE)

**Comment by SergeStinckwich** [comment]
@all-contributors please add @jigyasa-grover for code

**Comment by allcontributors** [comment]
@SergeStinckwich 

I've put up [a pull request](https://github.com/cormas/cormas/pull/233) to add @jigyasa-grover! :tada:

**Comment by jigyasa-grover** [comment]
> @all-contributors please add @jigyasa-grover for code

awesome, thank you @SergeStinckwich !

**Comment by allcontributors** [comment]
@jigyasa-grover 

I've put up [a pull request](https://github.com/cormas/cormas/pull/235) to add @jigyasa-grover! :tada:

**Comment by SergeStinckwich** [comment]
@all-contributors please add @peteruhnak for tool

**Comment by allcontributors** [comment]
@SergeStinckwich 

I've put up [a pull request](https://github.com/cormas/cormas/pull/245) to add @peteruhnak! :tada:

**Comment by SergeStinckwich** [comment]
@all-contributors please add @gabrami34 for test

**Comment by allcontributors** [comment]
@SergeStinckwich 

I've put up [a pull request](https://github.com/cormas/cormas/pull/379) to add @gabrami34! :tada:
